.github/actions/setup-rust/action.yml

```diff
@@ -41,12 +41,18 @@ outputs:
 runs:
   using: composite
   steps:
+    # Set steps.os.outputs.image to the specific OS (e.g. 'ubuntu20')
+    # see https://github.com/actions/cache/issues/543
+    - name: Get operating system name and version.
+      id: os
+      run: echo "::set-output name=image::$ImageOS"
+      shell: bash
```

Comment:
```suggestion
```

You can directly use `env.<variable_name>` see comment below

Reply:
7f7f1482873b86098b805c9b499fce615d613520